### PR TITLE
work folders added

### PR DESCRIPTION
**Pull request by yuusuf4real**
I added a template folder with different files such as index.html, style.css in css folder and script.js in the js folder

### CODE REVIEW

**Comment by abdulfataiaka** [comment]
The above PR content does not describe your changes.

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>mobaiz app</title>
+    <body>
+
```
Comment (abdulfataiaka) [inline public/index.html]: Why the extra lines below here?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>mobaiz app</title>
+    <body>
```
Comment (abdulfataiaka) [inline public/index.html]: Checkout online for how an HTML file should be structured at the base using `<html>`, `<head>`, and `<body>`

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+        <header>
+            <h1>Welcome to Mobaiz App</h1>
+            <p>the number one online shop for all kinds of mobile devices</p>
+        </header>
+
+    </body>
+    </head>
+</html>
```
Comment (abdulfataiaka) [inline public/index.html]: Add a new line

File: public/index.html (hunk 2 above)
Comment (yuusuf4real) [reply public/index.html]: Not having the head close tag above is a mistake

**Pull request merged**